html-tables/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Tables</title>
+  </head>
+
+  <body>
+    <h1>Student Grade Table</h1>
+    <div>
+      <table >
+        <thead>
+          <tr>
+            <th>Student Name</th>
+            <th>Course</th>
+            <th>Grade</th>
+          </tr>
+        </thead>
+        <tbody align="left">
+          <tr>
+            <th>Grace Hopper</th>
```

Comment:
Data in the table body should be in `td` elements, not `th` elements.

Reply:
Here are the changes Cody!

![image](https://user-images.githubusercontent.com/69921839/105112225-70805a00-5a77-11eb-9a14-68678c9bd7f6.png)

Reply:
Here are the changes I made!

![image](https://user-images.githubusercontent.com/69921839/105113267-6e1eff80-5a79-11eb-9650-135fd46b66bc.png)